scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/Task.java

```diff
@@ -380,6 +400,27 @@ public static Task parseJson(JSONObject jsonTask) {
         result.setTag(tag);
         result.setJobId(jobId);
         result.setJobName(jobName);
+
+        long startAtTime = 0L;
+        JSONObject genericInformationsObject = jsonTask.get("genericInformations").isObject();
+        if (genericInformationsObject != null) {
+            JSONValue genericInformationsValue = genericInformationsObject.get("START_AT");
+            if (genericInformationsValue != null) {
+                String genericInformationsStr = genericInformationsValue.isString().stringValue();
```

Comment:
I think scheduler core should be updated to embed directly the start at value in its local date. This way it is not longer to perform parsing at several sides. Besides, it will allow us to update the tasks filtering database query to filter tasks based on start_at value.